src/utils/Keep3rV2Job.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: BUSL 1.1
+pragma solidity 0.8.13;
+
+import "../interfaces/IKeep3rV2Job.sol";
+
+import "solmate/auth/authorities/MultiRolesAuthority.sol";
+import "keep3r/solidity/interfaces/IKeep3r.sol";
+
+abstract contract Keep3rV2Job is IKeep3rV2Job, MultiRolesAuthority {
```

Comment:
I don't think we need `MultiRolesAuthority` here, just a single admin address, a setter, and a modifier to check if the caller is admin will work throughout.

Reply:
ok, I'll make that change.